Extensions/PrefabExtensions.cs

```diff
@@ -0,0 +1,79 @@
+﻿using ProjectM;
```

Comment:
We shouldn't need this file. See the comments in the overview.

Reply:
Removed.